hackclub.com.yaml

```diff
@@ -220,6 +220,10 @@ alexa:
 - ttl: 1
   type: CNAME
   value: cname.vercel-dns.com.
+aloha:
+- ttl: 1
+  type: CNAME
+  value: https://aloha-college.com/en/aloha-college/.
```

Comment:
This CNAME is configured incorrectly and will not work, CNAMEs should not include https:// and should not have a path at the end

Reply:
Fixed @sampoder